lib/oelite/log.py

```diff
@@ -1,22 +0,0 @@
-import logging
```

Comment:
while I really miss a proper logging implementation in OE-lite, it is probably better to start by ripping away all the unused parts, before starting to implement new stuff. non-working features like this one adds to the confusion when reading through the code

It would be really nice to talk about how we could implement proper logging though instead of having to insert new print statements all-over the place every time we need to debug things :(